internal/worldstate/leveldb/commit_and_query_test.go

```diff
@@ -313,13 +313,18 @@ func TestCommitAndQuery(t *testing.T) {
 				require.NoError(t, err)
 				require.False(t, exist)
 			}
+
+			iter, err := l.GetIterator(db, "key0", "key3")
+			require.NoError(t, err)
+			require.NoError(t, iter.Error())
+			require.False(t, iter.Next())
```

Comment:
you need to Release this one too, right?

Reply:
Yes. Done